common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -57,6 +57,8 @@ public enum Feature {
 
         ADMIN2("New Admin Console", Type.DEFAULT, Feature.ADMIN_API),
 
+        LOGIN2("New Login Theme", Type.PREVIEW),
```

Comment:
I think we should probably default to `EXPERIMENTAL` here, considering this is very early code.